source/sdr.rst

```diff
@@ -104,21 +117,73 @@ Sediment Delivery Ratio
+        0.001, & \text{for } C<0.001\\
+        C,     & \text{otherwise}\\
+        \end{array}\right\}
+
 :math:`D_{up}` is the upslope component defined as:
 
-.. math:: D_{up}=\bar{C}\bar{S}\sqrt{A}
+.. math:: D_{up}=\bar{C}_{th}\bar{S}_{th}\sqrt{A}
     :label: d_up
 
-where :math:`\bar{C}` is the average :math:`C` factor of the upslope contributing area, :math:`S` is the average slope gradient of the upslope contributing area (:math:`m/m`) and :math:`A` is the upslope contributing area (:math:`m^2`). The upslope contributing area is delineated from a Multiple-Flow Direction algorithm.
+where :math:`\bar{C}_{th}` is the average thresholded :math:`C` factor of the upslope contributing area, :math:`\bar{S}_{th}` is the average thresholded slope gradient of the upslope contributing area (:math:`m/m`) and :math:`A` is the upslope contributing area (:math:`m^2`). The upslope contributing area is delineated from a Multiple-Flow Direction algorithm.
 
 The downslope component :math:`D_{dn}` is given by:
 
-.. math:: D_{dn}=\sum_i\frac{d_i}{C_i S_i}
+.. math:: D_{dn}=\sum_i\frac{d_i}{C_{th, i} S_{th,i}}
     :label: d_dn
 
-where :math:`d_i` is the length of the flow path along the ith cell according to the steepest downslope direction (:math:`m`) (see Figure 2), :math:`C_i` and :math:`S_i` are the :math:`C` factor and the slope gradient of the ith cell, respectively. Again, the downslope flow path is determined from a Multiple-Flow Direction algorithm.
+where :math:`d_i` is the length of the flow path along the ith cell according to the steepest downslope direction (:math:`m`) (see Figure 2), :math:`C_{th, i}` and :math:`S_{th, i}` are the thresholded cover-management factor and the thresholded slope gradient of the ith cell, respectively. Again, the downslope flow path is determined from a Multiple-Flow Direction algorithm.
+
+.. _pour points:
+
+Pour Points
++++++++++++
+
+:math:`d_i` is the distance that water travels downslope from the pixel before reaching a stream. Because SDR is calculated in terms of :math:`d_i`, it is only defined for pixels that drain into a stream on the map.
+
+In order to ensure that every pixel drains to a stream, and thus has defined :math:`d_i` and SDR values, the model adds pour points to the streams map if needed. A pour point is a point where water flows off the defined area of the flow direction raster, either off the edge or into a nodata area. If these points are not already marked as streams in the streams layer generated from flow accumulation (**flow_accum_streams.tif**) or the optional drainage layer input, the model will mark them as streams in **streams.tif**. These extra stream pixels may be a sign that something is wrong with your inputs:
+
+- If the pour point is outside your watersheds of interest, you may ignore it.
+
+- If the pour point is actually a stream in the real world, try increasing your TFA and/or using the optional drainage layer so that the model recognizes this stream.

```

Comment:
I'm confused here. I thought this was a list of how to deal with the model defining more pixels as streams than it should, so why would the user make a change "_so that the model recognizes this [more] stream_"? And increasing the TFA would lead the model to recognize fewer stream pixels anyway, not more.

Reply:
You're right it should say _decreasing_ TFA, not increasing, to make more streams. Hooray for reviews! 🎉 

My understanding is that, the user should provide data such that every pixel in their DEM drains into a stream.

When that doesn't happen, as of 3.9.1 the model will handle it by adding stream pixels on the pour points. But really, every pour point should _already_ be marked as a stream (either in the the stream layer that the model generates, and/or the optional streams input layer). 

Therefore, added stream pixels are a sign that you should mark more streams in that area, either by lowering the TFA or providing the optional streams input layer. Does that sound right?

Reply:
Ok, that makes more sense to me. @newtpatrol do you agree?